altair/api.py

```diff
@@ -233,6 +293,18 @@ def _shape_changed(self, name, old, new):
             self.shape._infer_type(self.parent.data)
 
 
+class Filter():
```

Comment:
We plan to release 0.8 (producing working vega 2 specs) some times next week (before [VIS2015](http://ieeevis.org/)).  Then shortly after, we will try to refactor and freeze our syntax in 0.9 to reduce the churn for you guys.  This might happen before VIS2015 but more likely shortly after the conference.  

Now, we also put our roadmap here: 
https://github.com/vega/vega-lite/wiki/Roadmap